<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimplePromptBox</class>
 <widget class="QDialog" name="SimplePromptBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimplePromptBox</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#SimplePromptBox
{
	border-radius:2px;
	background:rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="l_icon">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	min-width:16px;
	min-height:16px;
	max-width:16px;
	max-height:16px;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="te_content">
     <property name="styleSheet">
      <string notr="true">QTextEdit
{
	min-height:40px;
	border:none;
	font-size:14px;
	color:rgba(0,0,0,0.6500);
	background-color: rgba(255, 255, 255, 0);
}

</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btn_close">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton
{
	min-height:24px;
	min-width:24px;
	max-height:24px;
	max-width:24px;
	border-image:url(:/PromptBox/PromptBox/close_normal.png);
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources/CommonUtil.qrc"/>
 </resources>
 <connections/>
</ui>
